pymysql/connections.py

```diff
@@ -1191,4 +1227,70 @@ def _get_descriptions(self):
         assert eof_packet.is_eof_packet(), 'Protocol error, expecting EOF'
         self.description = tuple(description)
 
+
+class LoadLocalFile(object):
+    def __init__(self, filename, connection):
+        self.filename = filename
+        self.connection = connection
+
+    def _print_warnings(self):
+        self.connection._execute_command(COMMAND.COM_QUERY, 'SHOW WARNINGS')
+        warnings = MySQLResult(self.connection)
+        warnings.read()
```

Comment:
stdout may not valid file, or closed.
For example, print() may raise exception on mod_wsgi's default setting. (ref. https://code.google.com/p/modwsgi/wiki/ConfigurationDirectives#WSGIRestrictStdout)

I check how MySQLdb treat warnings.  It check MySQL warnings on cursor, and use `warnings.warn()` to show it.
https://github.com/farcepest/MySQLdb1/blob/d34fac681487541e4be07e6978e0db233faf8252/MySQLdb/cursors.py#L107-L120